### PR TITLE
Safari + small browser tweaks

### LINKED ISSUE #608: Landing page mid-size window quirk
Screen Shot 2021-04-15 at 1 45 26 PM

### LINKED ISSUE #619: Landing page bug in safari
Screen Shot 2021-04-15 at 10 36 56 PM

### PR DESCRIPTION
**Pull request by kkatzen**
closes #608
closes #619

### CODE REVIEW

**Comment by kkatzen** [comment]
Screen Shot 2021-04-23 at 11 03 23 PM
Screen Shot 2021-04-23 at 11 02 58 PM
Screen Shot 2021-04-23 at 11 02 47 PM
Screen Shot 2021-04-23 at 11 01 37 PM
Screen Shot 2021-04-23 at 11 01 24 PM
Screen Shot 2021-04-23 at 11 00 36 PM
Screen Shot 2021-04-23 at 10 59 59 PM
Screen Shot 2021-04-23 at 10 58 40 PM
Screen Shot 2021-04-23 at 10 58 29 PM

File: frontend/src/pages/AboutUs/ContactUsTab.tsx [hunk 1]
```diff
@@ -15,41 +15,44 @@ function ContactUsTab() {
...
+              />
+            </Grid>
+          </Hidden>
         </Grid>
-        
+
         <Grid
           container
           className={styles.GridOutlinedRow}
-          direction="column"
```
Comment (efregoso) [inline frontend/src/pages/AboutUs/ContactUsTab.tsx:52]: JW, what was the effect of keeping this?

File: frontend/src/pages/AboutUs/TheProjectTab.tsx [hunk 2]
```diff
@@ -7,6 +7,45 @@ import {
   LinkWithStickyParams,
   EXPLORE_DATA_PAGE_LINK,
 } from "../../utils/urlutils";
+import Hidden from "@material-ui/core/Hidden";
+
+function ChartGif(props: {
```
Comment (efregoso) [inline frontend/src/pages/AboutUs/TheProjectTab.tsx:12]: Maybe change function name to "AimToGoItem"?

File: frontend/src/pages/Landing/LandingPage.tsx [hunk 3]
```diff
@@ -16,6 +16,29 @@ import {
 } from "../../utils/urlutils";
 import { ABOUT_US_FAQ_TAB_INDEX } from "../AboutUs/AboutUsPage";
 
+function ChartGif(props: { src: string; alt: string; text: string }) {
```
Comment (efregoso) [inline frontend/src/pages/Landing/LandingPage.tsx:19]: "TakeALookAroundItem"?

**Review by efregoso — APPROVED** [review]

**Pull request merged**